### PR TITLE
1460 add living style guide

### PR DESCRIPTION
**Pull request by ilovethebends**
# What's in this PR?

List the changes you made and your reasons for them.

Make sure any changes to code include changes to documentation.

## References
Fixes #

Progress on: #

### CODE REVIEW

File: lib/styleguide/app/routes/styleguide/components/svg/sprite-icon.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
```
Comment (snewcomer) [inline lib/styleguide/app/routes/styleguide/components/svg/sprite-icon.js]: here it could be

`import Route from "@ember/routing/route"`

**Comment by JoshSmith** [comment]
Hey @ilovethebends just checking in. Anything you need from us at all?

**Comment by ilovethebends** [comment]
I was looking for styles I could reuse to build the components menu. 
I was looking for vertical menus through out the site and didn't find anything.

**Comment by JoshSmith** [comment]
Instead of doing a left navigation menu for the components (especially since many are full-width for the page), how about doing an `ember-power-select` that contains the list of components you want to render?

**Comment by ilovethebends** [comment]
@JoshSmith Were you thinking of individual routes for each component?
or should we just render the component on the same page?

**Comment by ilovethebends** [comment]
@JoshSmith 
So far I Added a few simple components
- animated high five
- categories-list
- donation-card-item
- sprite-icon

**Comment by JoshSmith** [comment]
I think this is in a state where it's basically reviewable. I could use some thoughts from you on this @begedin.

**Comment by joshsmith** [comment]
@begedin can you share thoughts on this?

File: lib/styleguide/app/initializers/styleguide-routing.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import styleguideRoutes from '../styleguide-routes';
+
+export function initialize() {
+  MyAppRouter.map(styleguideRoutes);
+}
+
+export default {
+  name: 'styleguide-routes',
+  initialize
+};
```
Comment (begedin) [inline lib/styleguide/app/initializers/styleguide-routing.js]: Probably should change the wording from `MyApp` to `CodeCorpsEmber`.

Also the comment on line 1

**Comment by joshsmith** [comment]
@ilovethebends were you wanting to close this now in favor of #1565?

I know we discuss some trouble there with component availability across engines.

**Pull request closed without merging**